### DISCUSSION TITLE
Implications of the letter X in "MNX"

### DISCUSSION THREAD

**Discussion by samuelbradshaw** [w3c/mnx#303] [General]
The letters M and N in "MNX" stand for "music notation," but it's never been clear to me what the X stands for.

However, the letter X brings to mind a few things that don't represent MNX very well:

1. XML, XPath: MNX is no longer an XML format.
2. Exchange, interchange: MusicXML was an interchange format, and MNX will be too. But unlike MusicXML, MXL aspires to go beyond that, and be suitable as a native file format. Thinking of it as primarily an exchange format holds it back.
3. Windows XP, Mac OS X, ActiveX, and other brand names introduced around the turn of the century that used "X" to indicate that they were new and modern (but they no longer are).

I propose that with the change from XML to JSON, we should also consider giving MNX a new name. As of right now I don't have a proposed name, but I'm curious if others agree.

Potential criteria:
- Easy to say and remember
- Should bring to mind music and/or music notation
- Shouldn't bring to mind XML
- Shouldn't bring to mind previous-generation technology
- Shouldn't be too similar to other music format names (MusicXML, MEI, etc.)
- Might be three or four letters like most file extensions, but doesn't need to be
- Bonus: Might have a long and short variant for a compressed format (similar to MusicXML –> MXL)

**Comment by mscuthbert** [comment]
To me the nx implies Next

**Reply by webern** [reply]
X is often used for eXchange, as in Music Notation Exchange.

**Comment by adrianholovaty** [comment]
We've already renamed from MNX-Common to MNX back in 2020 (see issue #191), so it would take a pretty strong argument to convince me to rename yet again.

IIRC, the original naming was intended to suggest "Music Notation neXt" as opposed to XML.

**Reply by ahankinson** [reply]
"MNX stands for "Music Notation X", where "X" suggests "XML", "eXtended", and potentially other X-related things as may come to mind."

Section 1.1: https://w3c.github.io/mnx/overview/

I remember in initial discussions that the "X" was informally also for "Unknown" since it was unclear what the relationship between MNX and MusicXML was going to be -- whether it was exploratory work that was going to be folded back into MusicXML, or a thing all its own.

**Reply by mscuthbert** [reply]
And even if it did originally signify XML, AJAX (Asynchronous Javascript And <s>XML</s> JSON) shows that the switch can happen without major complications.

As it is, there's a W3C standard, etc. which isn't nothing to change.

**Comment by adrianholovaty** [comment]
We discussed this in today's biweekly co-chairs meeting, and it's safe to say we won't be changing the name of the format again. Marking as closed.
